recipes/openjdk/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class OpenJDK(ConanFile):
+    name = "openjdk"
+    url = "https://github.com/conan-io/conan-center-index/"
+    description = "Java Development Kit builds, from Oracle"
+    homepage = "https://jdk.java.net"
+    license = "https://openjdk.java.net/legal/gplv2+ce.html"
```

Comment:
https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html

```suggestion
    license = "GPL-2.0-with-classpath-exception"
```